### DISCUSSION TITLE
Missing `redirect()->intended(...);` and `back` equivalent for redirections with "navigate"

### DISCUSSION THREAD

**Discussion by nunomaduro** [livewire/livewire#6797] [7. Bugs (archived)]
### Livewire version

v3.0.3

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

At this moment, there is the following "redirect" methods available if we were to use "wire:navigate" on redirections:

```
$this->redirect(..., navigate: true);
$this->redirectRoute(..., navigate: true);
$this->redirectAction(..., navigate: true);
```

However, it seems to be missing direct equivalent to `redirect()->intended(...);` and `back()`.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by robsontenorio** [comment]
@nunomaduro do you have any workaround for now?

**Comment by gazzoy** [comment]
+1

**Comment by patie** [comment]
anybody? 🙏

**Reply by danie-ramdhani** [reply]
it's already implemented 😅 https://livewire.laravel.com/docs/redirecting#redirect-to-intended

**Reply by patie** [reply]
Thx, but its not working with Laravel Filament in SPA mode for some reason, but probably its question for Filament repo
